metrics/grafana/tikv_details.json

```diff
@@ -30316,7 +30340,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "tikv_engine_num_snapshots{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", db=\"$db\"}",
+              "expr": "10 * rate(tikv_engine_num_snapshots_sum{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", db=\"$db\"}[1m])",
```

Comment:
why plus 10

Reply:
Because we report the statistics each 10s. So the rate needs to be adjusted x10.